pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -195,6 +195,10 @@ def publish(self, topic, data, **attrs):
 
         # Create the Pub/Sub message object.
         message = types.PubsubMessage(data=data, attributes=attrs)
+        if message.ByteSize() > self.batch_settings.max_bytes:
+            raise ValueError('Message being published is too large for the '
+                             'batch settings with max bytes {}.'.
```

Comment:
Nit: please don't do hanging indents. Please line break after the opening `(` and indent once.

Reply:
I'd like to suggest to just leave it since the other two TypeErrors are the same way...

Reply:
Please don't. :)